core/src/main/java/jenkins/management/AdministrativeMonitorsApi.java

```diff
@@ -0,0 +1,62 @@
+package jenkins.management;
+
+import hudson.Extension;
+import hudson.model.AdministrativeMonitor;
+import hudson.model.PageDecorator;
+import hudson.model.RootAction;
+import jenkins.model.Jenkins;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.StaplerRequest;
+import org.kohsuke.stapler.StaplerResponse;
+import org.kohsuke.stapler.verb.GET;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Collections;
+
+@Extension
+@Restricted(NoExternalUse.class)
+public class AdministrativeMonitorsApi implements RootAction {
+    @GET
```

Comment:
Should these calls have explicit permission checks?

Reply:
The authorization is implicit through the decorator. That said I think it would make sense to have more explicit auth. I have no idea how to got about this. If anyone wants to commit straight to my branch please go ahead :D

Reply:
@daniel-beck @Wadeck  how would you go about this?

Reply:
it's quite common to rely on permission checks in this way as far as I know

Reply:
In this case it's a bit special because the permission is checked deeper in the code. It was recently changed from Admin to SystemRead. It means that if we have an additional check here, it's a bit a duplicate code, so I am fine with the current approach.

Of course if the method is starting to do something else, the explicit permission will be required, but we should not be future-proof without good rational.